adblockparser/parser.py

```diff
@@ -291,7 +291,7 @@ def __init__(self, rules, supported_options=None, skip_unsupported_rules=True,
         self.rules = [
             r for r in (
                 r if isinstance(r, rule_cls) else rule_cls(r)
-                for r in rules
+                for r in rules if r
```

Comment:
I think we should strip whitespaces before checking if rule text is empty: AdblockRule class strips whitespaces, it makes sense to be consistent with it.